### PR TITLE
Allow specify package versions

### PR DESCRIPTION
**Pull request by elfosardo**
With this change we can specify package versions to be installed
directly in the packages list (main or extra).

### CODE REVIEW

**Review by iurygregory — APPROVED** [review]

File: prepare-image.sh [hunk 1]
```diff
@@ -5,10 +5,10 @@ set -euxo pipefail
 dnf install -y python3 python3-requests
 curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python3 - -b master current-tripleo
 dnf upgrade -y
-dnf --setopt=install_weak_deps=False install -y $(cat /tmp/${PKGS_LIST})
+xargs -rd'\n' dnf --setopt=install_weak_deps=False install -y < /tmp/${PKGS_LIST}
```
Comment (derekhiggins) [inline prepare-image.sh:8]: Can we add a "-t" to the xargs command so that the dnf command that ran is logged (and below)

Comment (elfosardo) [reply prepare-image.sh:8]: yes, we can! good idea :)

**Review by dtantsur — APPROVED** [review]

**Pull request merged**